grid/recruitment/list-candidates/maps/greenhouse.suma

```diff
@@ -0,0 +1,49 @@
+profile = "recruitment/list-candidates@1.0"
+provider = "greenhouse"
+
+map ListCandidates {
+  http GET "/v1/candidates" {
+    security "apiKeyBasicAuth"
+    
+
+    request {
+      query {
+        job_id = input.jobId
+        page = input.page
+        per_page = input.limit
+      }
+    }
+
+
+    response 200 "application/json" {
+      map result {
+        candidates = body.map(candidate => {
+        const name = `${candidate.first_name} ${candidate.last_name}`
```

Comment:
nitpick: there should be 2 more spaces at the beginning of this, and the following lines, so that the block is correctly indented

(hopefully we should have an autoformatter soon 🙏 )